src/context.lisp

```diff
@@ -110,6 +112,9 @@
                             (declare (ignore _))
                             (ssl-ctx-free ctx))))
       (ssl-ctx-set-options ctx (apply #'logior (append disabled-protocols options)))
+      (ssl-ctx-set-min-proto-version ctx min-ssl-version)
+      (ssl-ctx-set-max-proto-version ctx max-ssl-version)
+      (ssl-ctx-set-min-proto-version ctx +tls1-2-version+)
```

Comment:
hardcoded value? is that a mistake?

Reply:
Yes, it's something I left in from testing. The first two lines (apply value from args) are correct, the third needs to be removed. I'll do that.